css/src/components/table.css

```diff
@@ -0,0 +1,168 @@
+.yoast table {
+	font-size: var(--font-size-default);
```

Comment:
I see all the CSS variables in this PR still miss the `--yoast-` prefix. Should it be addressed now or later?

Reply:
This will be fixed in a separate issue: https://yoast.atlassian.net/browse/FRO-102